pkg/cmd/task/list.go

```diff
@@ -127,12 +127,4 @@ func listAllTasks(cs versioned.Interface, ns string) (*v1alpha1.TaskList, error)
 	return tasks, nil
 }
 
-func listTaskDetails(cs versioned.Interface, ns string) (*v1alpha1.TaskList, error) {
```

Comment:
This was duplicated ? :thinking:

Reply:
not exactly, but more or less? :thinking:  Talking to @sthaha , couldn't figure out why `listTaskDetails` was necessary.